### PR TITLE
i.modis manual: add missing products

### PR DESCRIPTION
**Pull request by neteler**
Various MODIS products were missing from the manual page.
Added now.

### CODE REVIEW

File: grass7/imagery/i.modis/i.modis.html [hunk 1]
```diff
@@ -54,16 +54,41 @@ <h2>DESCRIPTION</h2>
 <h2>Supported MODIS products</h2>
 These products are currently supported:
 
+<h3>MODIS Surface Reflectance</h3>
+<ul>
+  <li><b>Surface spectral reflectance daily 1 km and 500 m (Terra/Aqua)</b>: Bands 1 through 7
+      corrected for atmospheric conditions such as gasses, aerosols, and Rayleigh
+      scattering in Sinusoidal projection daily. Provided along with the 500 meter (m)
+      surface reflectance, observation, and quality bands are a set of ten 1 kilometer
+      observation bands and geolocation flags. The reflectance layers from the MYD09GA
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:63]:
```suggestion
      observation bands and geolocation flags. The reflectance layers from MOD09GA and MYD09GA
```

File: grass7/imagery/i.modis/i.modis.html [hunk 2]
```diff
@@ -54,16 +54,41 @@ <h2>DESCRIPTION</h2>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod09gav061/">MOD09GA</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd09gav061/">MYD09GA</a>
+      product pages)</li>
+  <li><b>Surface spectral reflectance eight day 500 m (Terra/Aqua)</b>: Bands 1 through 7
+      corrected for atmospheric conditions such as gasses, aerosols, and Rayleigh
+      scattering in Sinusoidal projection during an 8-day period. Along with the seven
+      500 meter (m) reflectance bands are two quality layers and four observation bands.
+      For each pixel, a value is selected from all the acquisitions within the 8-day
+      composite period. The criteria for the pixel choice include cloud and solar
+      zenith. When several acquisitions meet the criteria the pixel with the minimum
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:74]:
```suggestion
      zenith. When several acquisitions meet the criteria, the pixel with the minimum
```

**Review by veroandreo — APPROVED** [review]

File: grass7/imagery/i.modis/i.modis.html (hunk 1 above)
Comment (neteler) [reply grass7/imagery/i.modis/i.modis.html:63]: This is the original LPDAAC text...

File: grass7/imagery/i.modis/i.modis.html (hunk 2 above)
Comment (neteler) [reply grass7/imagery/i.modis/i.modis.html:74]: This is the original LPDAAC text...

File: grass7/imagery/i.modis/i.modis.html [hunk 3]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
 These products are currently supported:
 
+<h3>MODIS Surface Reflectance</h3>
+<ul>
+  <li><b>Surface spectral reflectance daily 1 km and 500 m (Terra/Aqua)</b>: Bands 1 through 7
+      corrected for atmospheric conditions such as gasses, aerosols, and Rayleigh
+      scattering in Sinusoidal projection daily. Provided along with the 500 meter (m)
+      surface reflectance, observation, and quality bands are a set of ten 1 kilometer
+      observation bands and geolocation flags. The reflectance layers from MOD09GA and MYD09GA
+      are used as the source data for many of the MODIS land products.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:64]:
```suggestion
      are used as the source data for many of the MODIS land products 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 4]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod09gav006/">MOD09GA</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd09gav006/">MYD09GA</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:67]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 5]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+  <li><b>Surface spectral reflectance eight day 500 m (Terra/Aqua)</b>: Bands 1 through 7
+      corrected for atmospheric conditions such as gasses, aerosols, and Rayleigh
+      scattering in Sinusoidal projection during an 8-day period. Along with the seven
+      500 meter (m) reflectance bands are two quality layers and four observation bands.
+      For each pixel, a value is selected from all the acquisitions within the 8-day
+      composite period. The criteria for the pixel choice include cloud and solar
+      zenith. When several acquisitions meet the criteria, the pixel with the minimum
+      channel 3 (blue) value is used.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:75]:
```suggestion
      channel 3 (blue) value is used 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 6]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod09a1v006/">MOD09A1</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd09a1v006/">MYD09A1</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:78]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 7]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+</ul>
+
+
 <h3>MODIS LST - Land Surface Temperature</h3>
 <ul>
-  <li><b>Land Surface Temperature daily 1 Km (Terra/Aqua)</b>: product provides per-pixel temperature
+  <li><b>Land Surface Temperature daily 1 km (Terra/Aqua)</b>: product provides per-pixel temperature
       and emissivity values in a sequence of swath-based to grid-based global products in Sinusoidal projection.
       The MODIS/Terra-Aqua LST/E Daily L3 Global 1 km Grid product (MOD11A1/MYD11A1), is tile-based
       and gridded in the Sinusoidal projection, and produced daily at 1 km spatial resolution.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:87]:
```suggestion
      and gridded in the Sinusoidal projection, and produced daily at 1 km spatial resolution 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 8]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
       (related <a href="https://lpdaac.usgs.gov/products/mod11a1v006/">MOD11A1</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd11a1v006/">MYD11A1</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd11a1v006/">MYD11A1</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:90]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 9]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+  <li><b>Land Surface Temperature eight day 1 km (Terra/Aqua)</b>: data are composed from the daily
       1-kilometer LST product (MOD11A1/MYD11A1) and stored on a 1-km Sinusoidal grid as the
       average values of clear-sky LSTs during an 8-day period.<br>
       MOD11A2/MYD11A2 is comprised of daytime and nighttime LSTs, quality assessment,
       observation times, view angles, bits of clear sky days and nights, and
       emissivities estimated in Bands 31 and 32 from land cover types
       (related <a href="https://lpdaac.usgs.gov/products/mod11a2v006/">MOD11A2</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd11a2v006/">MYD11A2</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd11a2v006/">MYD11A2</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:99]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 10]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+  <li><b>Land Surface Temperature eight day ~6 km (Terra/Aqua)</b>: products provide per-pixel
       temperature and emissivity values in a sequence of swath-based to grid-based
-      global products. The MODIS/Terra-Aqua LST/E Daily L3 Global 6 Km Grid
+      global products. The MODIS/Terra-Aqua LST/E Daily L3 Global 6 km Grid
       (Short name: MOD11B1/MYD11B1), is tile-based and gridded in the Sinusoidal projection,
       and produced daily at 5600m spatial resolution
       (related <a href="https://lpdaac.usgs.gov/products/mod11b1v006/">MOD11B1</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd11b1v006/">MYD11B1</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd11b1v006/">MYD11B1</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:107]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 11]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+      products with a pixel size of 5,600 meters. Each LST&amp;E pixel value in the MOD11B3
+      is a simple average of all the corresponding values from the MOD11B1 collected during
+      the month period. Each MOD11B3 granule consists of 19 layers including daytime and
+      nighttime layers for LSTs, quality control assessments, observation times, view
+      zenith angles, and number of clear sky observations along with percentage of land
+      in the tile and emissivities from bands 20, 22, 23, 29, 31, and 32. Unique to the
+      MOD11B products are additional day and night LST layers generated from band 31 of
+      the corresponding 1 km
+      <a href="https://lpdaac.usgs.gov/products/mod11_l2v006/">MOD11_L2 swath product</a>
+      aggregated to the 6 km grid.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:119]:
```suggestion
      aggregated to the 6 km grid 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 12]
```diff
@@ -54,31 +54,71 @@ <h2>DESCRIPTION</h2>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod11b3v006/">MOD11B3</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd11b3v006/">MYD11B3</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:122]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 13]
```diff
@@ -104,7 +144,7 @@ <h3>MODIS VI - Vegetation Indices</h3>
       blue band, the EVI algorithm uses the 500m blue band to correct for residual
       atmospheric effects, with negligible spatial artifacts
       (related <a href="https://lpdaac.usgs.gov/products/mod13q1v006/">MOD13Q1</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd13q1v006/">MYD13Q1</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd13q1v006/">MYD13Q1</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:148]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 14]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
       be used for characterizing land surface biophysical properties and processes,
       including primary production and land cover conversion
       (related <a href="https://lpdaac.usgs.gov/products/mod13a1v006/">MOD13A1</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd13a1v006/">MYD13A1</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd13a1v006/">MYD13A1</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:174]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 15]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
+      Normalized Difference Vegetation Index (NDVI) which is referred to as the continuity
+      index to the existing National Oceanic and Atmospheric Administration-Advanced
+      Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The second vegetation
+      layer is the Enhanced Vegetation Index (EVI), which has improved sensitivity
+      over high biomass regions. The algorithm chooses the best available pixel value
+      from all the acquisitions from the 16 day period. The criteria used is low
+      clouds, low view angle, and the highest NDVI/EVI value. Along with the vegetation
+      layers and the two quality layers, the HDF file will have MODIS reflectance
+      bands 1 (red), 2 (near-infrared), 3 (blue), and 7 (mid-infrared), as well as
+      four observation layers.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:186]:
```suggestion
      four observation layers 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 16]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod13q1v006/">MOD13Q1</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd13q1v006/">MYD13Q1</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:189]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 17]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
-  <li><b>VI sixteen days 1 Km (Terra/Aqua)</b>:
+  <li><b>VI sixteen days 1 km (Terra/Aqua)</b>:
       The MOD13A2 product provides a Vegetation Index (VI) value at a per pixel basis. There
       are 2 primary vegetation layers. The first is the Normalized Difference Vegetation Index (NDVI)
       which is referred to as the continuity index to the existing National Oceanic and Atmospheric
       Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The second
       vegetation layer is the Enhanced Vegetation Index (EVI), which has improved sensitivity over
       high biomass regions. The data are provided at 1000 m resolution as a gridded level-3 product
-      in the Sinusoidal projection
+      in the Sinusoidal projection.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:197]:
```suggestion
      in the Sinusoidal projection 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 18]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
       (related <a href="https://lpdaac.usgs.gov/products/mod13a2v006/">MOD13A2</a> and
-      <a href="https://lpdaac.usgs.gov/products/myd13a2v006/">MYD13A2</a> 
+      <a href="https://lpdaac.usgs.gov/products/myd13a2v006/">MYD13A2</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:200]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 19]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
+  <li><b>VI monthly 1 km (Terra/Aqua)</b>:
+      The MOD13A3 product provides a Vegetation Index (VI) value at a per pixel basis. There
+      are 2 primary vegetation layers. The first is the Normalized Difference Vegetation Index (NDVI)
+      which is referred to as the continuity index to the existing National Oceanic and Atmospheric
+      Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The second
+      vegetation layer is the Enhanced Vegetation Index (EVI), which has improved sensitivity over
+      high biomass regions. The data are provided at 1000 m resolution as a gridded level-3 product
+      in the Sinusoidal projection. Provided along with the vegetation layers and the two quality
+      assurance (QA) layers are reflectance bands 1 (red), 2 (near-infrared), 3 (blue), and 7
+      (mid-infrared), as well as three observation layers.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:210]:
```suggestion
      (mid-infrared), as well as three observation layers 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 20]
```diff
@@ -130,18 +170,46 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
+      (related <a href="https://lpdaac.usgs.gov/products/mod13a3v006/">MOD13A3</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd13a3v006/">MYD13A3</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:213]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 21]
```diff
@@ -151,47 +219,25 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
-      the year and extend into the next year. The product can be produced with
-      two to eight days of input. There may not always be eight days of input,
-      because of various reasons, so the user should check the attributes to
-      determine on what days observations were obtained. See the validation webpage
-      for details on the validation and validation definitions
-      (related <a href="https://nsidc.org/data/MOD10A2">MOD10A2</a> and
-      <a href="https://nsidc.org/data/MYD10A2">MYD10A2</a> 
+      (related <a href="https://lpdaac.usgs.gov/products/mod13c1v006/">MOD13C1</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd13c1v006/">MYD13C1</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:224]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 22]
```diff
@@ -151,47 +219,25 @@ <h3>MODIS VI - Vegetation Indices</h3>
...
+      The MOD13C2 product provides a Vegetation Index (VI) value at a per pixel basis. There
+      are 2 primary vegetation layers. The first is the Normalized Difference Vegetation Index (NDVI)
+      which is referred to as the continuity index to the existing National Oceanic and Atmospheric
+      Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The second
+      vegetation layer is the Enhanced Vegetation Index (EVI), which has improved sensitivity over
+      high biomass regions. The Climate Modeling Grid (CMG) consists 3600 rows and 7200 columns
+      of 5600 m pixels and is provided as a global latitude/longitude grid
+      (related <a href="https://lpdaac.usgs.gov/products/mod13c2v006/">MOD13C2</a> and
+      <a href="https://lpdaac.usgs.gov/products/myd13c2v006/">MYD13C2</a>
       product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:235]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 23]
```diff
@@ -214,6 +260,59 @@ <h3>MODIS AOD - Aerosol Optical Depth</h3>
...
+      sinusoidal projection. Tiles are approximately 1200 x 1200 km in area. A
+      single scientific data set (SDS) of snow cover and a single SDS of QA data
+      along with local and global attributes comprise the data product file. The
+      daily level 3 snow product is the result of selecting an observation from
+      the multiple observations mapped to a cell of the MOD10_L2G (or MYD10_L2G)
+      product. See the validation webpage for details on the validation and
+      validation definitions
+      (related <a href="https://nsidc.org/data/MOD10A1">MOD10A1</a> and
+      <a href="https://nsidc.org/data/MYD10A1">MYD10A1</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:280]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 24]
```diff
@@ -214,6 +260,59 @@ <h3>MODIS AOD - Aerosol Optical Depth</h3>
...
+      days is mapped as maximum snow extent in one SDS and as a chronology of
+      observations in the other SDS. Eight-day periods begin on the first day of
+      the year and extend into the next year. The product can be produced with
+      two to eight days of input. There may not always be eight days of input,
+      because of various reasons, so the user should check the attributes to
+      determine on what days observations were obtained. See the validation webpage
+      for details on the validation and validation definitions
+      (related <a href="https://nsidc.org/data/MOD10A2">MOD10A2</a> and
+      <a href="https://nsidc.org/data/MYD10A2">MYD10A2</a>
+      product pages)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:294]:
```suggestion
      product pages).</li>
```

File: grass7/imagery/i.modis/i.modis.html [hunk 25]
```diff
@@ -214,6 +260,59 @@ <h3>MODIS AOD - Aerosol Optical Depth</h3>
...
+  map of surface water at 250 meter (m) spatial resolution. The data are available annually
+  from 2000 to 2015. MOD44W Version 6 is derived using a decision tree classifier trained
+  with MODIS data and validated with the Version 5 MOD44W data product. A series of masks
+  are applied to address known issues caused by terrain shadow, burn scars, cloudiness,
+  or ice cover in oceans. A primary improvement in Version 6 is the generation of time
+  series data rather than a simple static representation of water, given that water
+  bodies fluctuate in size and location over time due to both natural and anthropogenic
+  causes. Provided in each MOD44W Version 6 Hierarchical Data Format 4 (HDF4) file are
+  layers for land, water, no data, and an associated per pixel quality assurance (QA)
+  layer that provides users with information on the determination of water.
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:310]:
```suggestion
  layer that provides users with information on the determination of water 
```

File: grass7/imagery/i.modis/i.modis.html [hunk 26]
```diff
@@ -214,6 +260,59 @@ <h3>MODIS AOD - Aerosol Optical Depth</h3>
...
+  (related <a href="https://lpdaac.usgs.gov/products/mod44wv006/">MOD44W</a>
+  product page)</li>
```
Comment (veroandreo) [inline grass7/imagery/i.modis/i.modis.html:312]:
```suggestion
  product page).</li>
```

**Review by veroandreo — APPROVED** [review]
I tried to homogenize the use of dots only

**Pull request merged**